pkg/printers/internalversion/printers.go

```diff
@@ -18,6 +18,7 @@ func AddHandlers(h printers.PrintHandler) {
 		{Name: "Version", Type: "string", Description: "KubernetesVersion represents version of the member cluster."},
 		{Name: "Mode", Type: "string", Description: "SyncMode describes how a cluster sync resources from karmada control plane."},
 		{Name: "Ready", Type: "string", Description: "The aggregate readiness state of this cluster for accepting workloads."},
+		{Name: "MetadataSufficient", Type: "string", Description: "The status indicate that all APIs are obtained"},
```

Comment:
Is this used to format the output for `kubectl get` ?

Is there a test report?